examples/newrelic-express-apm/src/sdk-v3/sdk.js

```diff
@@ -31,7 +31,11 @@ const httpMiddlewareOptions = {
 // newrelic options
 const telemetryOptions = {
   createTelemetryMiddleware,
+  userAgent: 'typescript-sdk-middleware-newrelic',
   apm: () => agent,
+  customMetrics: {
```

Comment:
This option is not described in the docs: https://docs.commercetools.com/sdk/observability/newrelic. Please add it with explanation what it means.

Reply:
Yes, it's included in the branch where I am working on ;) 

Please check https://github.com/commercetools/commercetools-docs/tree/update-ts-docs